styles/projects.css

```diff
@@ -0,0 +1,72 @@
+/* Works section */
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  row-gap: 5.5rem;
+  padding: 7.125rem 1.5rem;
+}
+
+.projects {
+  display: flex;
+  flex-direction: column;
+  gap: 0.75rem;
+  min-height: 100vh;
+  border: 0.0625rem solid #dfe1e6;
+  border-radius: 1rem;
+  background-color: #fff;
+  padding: 1rem;
+}
```

Comment:
Please know that each project card is too long, please check the screenshots below, the left one is from your website and the one on the right is the one from the Figma design:

<div>
  <img src="https://user-images.githubusercontent.com/20567503/176566379-7b854994-a03d-412c-a0a0-d8c796d7b4e5.png" width="200" />
  <img src="https://user-images.githubusercontent.com/20567503/176566409-54bbc7b9-f84b-44d0-b18a-80c3765357bf.png" width="200" />
</div>

Kindly follow the design in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to fix this. 😁😁

(Hint: You can remove the `min-height` you set and that should fix it). 😉